cmd/docker/docker.go

```diff
@@ -155,7 +156,7 @@ func main() {
 	stdin, stdout, stderr := term.StdStreams()
 	logrus.SetOutput(stderr)
 
-	dockerCli := command.NewDockerCli(stdin, stdout, stderr)
+	dockerCli := command.NewDockerCli(stdin, stdout, stderr, isContentTrustEnabled())
```

Comment:
Do we want to have that in the initializer prototype? rather with a dependency injection or just done from within the initializer function without modifying the prototype?

Reply:
I'm ok with it either way

Reply:
Was also looking at this a bit (not a fan of boolean args), but perhaps that's just a nit